backend/fms_core/viewsets/_utils.py

```diff
@@ -378,8 +378,15 @@ def default_batch_prefilling(sample_id_list: List[int], template, batch_sheet_na
 
             return batch_rows_list
 
+        def get_step_specific_experiment_prefix(step_name: str):
+            match step_name:
+                case "Experiment Run Infinium":
+                    return ""
+                case _:
+                    return "CHANGE ME "
```

Comment:
Oh my, a match expression where you would normally use an if-else statement 🤭.

Reply:
Leaving some space for other experiment run to have their own pattern.